tests/distributed/conftest.py

```diff
@@ -0,0 +1,24 @@
+# content of conftest.py
+
+import pytest
+
+
+def pytest_addoption(parser):
```

Comment:
I don't think you need this twice, can probably just add in a conftest.py in tests rather than in both utils and distributed